src/containers/App.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import Inputs from '../components/Inputs';
+import Outputs from '../components/Outputs';
+
+function App() {
+
+  const [amount, setAmount] = useState(0);
+  const [result, setResult] = useState(0);
+
+  return (
+    <main className="app">
+      <h1 className="app__title">Tip Amount Calculator</h1>
+      <Inputs amount={amount} setAmount={setAmount} setResult={setResult} />
+      <Outputs amount={amount} result={result} />
```

Comment:
"Inputs" and "Outputs" are fairly good names, but it is possible to make them more specific? Good naming helps other coders to be able to more easily walk through your code.